src/code_800029B0.c

```diff
@@ -97,17 +97,9 @@ u16 D_800DC5FC = 0;
 u8 *D_800DC600 = &osAppNmiBuffer[0];
 s32 gIsMirrorMode = 0;
 f32 D_800DC608 =  1.0f;
-Lights1 D_800DC610[] = {
-    {
-        {175, 175, 175, 0, 175, 175, 175, 0},
-        {255, 255, 255, 0, 255, 255, 255, 0, 0, 0, 120, 0},
-    },
-
-    {
-        {115, 115, 115, 0, 115, 115, 115, 0},
-        {255, 255, 255, 0, 255, 255, 255, 0, 0, 0, 120, 0},
-
-    },
+Lights1 D_800DC610[2] = {
```

Comment:
Should sizes be hard-coded like this?

Reply:
In this specific case I see no harm in it, but I can remove it if desired

Reply:
I think remove it.